src/containers/Forms/Teacher/EditTeacherForm.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router';
+import TeacherForm from './TeacherForm';
+import { useDispatch, useSelector } from 'react-redux';
+import { editTeacher, getTeacherById } from '../../../store/actions/teachersActions';
+
+const EditTeacherForm = () => {
+  const dispatch = useDispatch();
+  const teacher = useSelector((state) => state.teachers.teacher);
+  const params = useParams();
+  const paramsId = params.id;
+
+  useEffect(() => {
+    dispatch(getTeacherById(paramsId));
+  }, [dispatch, paramsId]);
+
+  const updateTeacher = (teacherForUpdate, id) => {
```

Comment:
Отличается с createTeacherForm
```suggestion
  const updateTeacherHandler = (teacherForUpdate, id) => {
```